src/handlebars/about.handlebars

```diff
@@ -16,9 +16,12 @@
         </p>
 
         <p>
-          AdoptOpenJDK provides rock-solid OpenJDK binaries for the
-          Java ecosystem and also provides <a target="_blank" href="https://www.github.com/AdoptOpenJDK">infrastructure as code</a>,
-          and a <a href="https://ci.adoptopenjdk.net">Build farm</a> for builders of OpenJDK, on any platform.
+          AdoptOpenJDK provides rock-solid <a href="https://openjdk.java.net" target="_blank">OpenJDK</a> and
+          <a href="https://www.eclipse.org/openj9/" target="_blank">Eclipse OpenJ9</a> (with OpenJDK class libraries)
+          binaries for the Java ecosystem and also provides
+          <a target="_blank" href="https://www.github.com/AdoptOpenJDK">infrastructure as code</a>,
+          and a <a href="https://ci.adoptopenjdk.net">Build farm</a> for builders of OpenJDK and Eclipse OpenJ9
+          (with OpenJDK class libraries), on any platform.
```

Comment:
do we need to mention `(with OpenJDK class libraries)` 2 times?

Reply:
We do in this case yes